src/Whim.Runner/App.xaml

```diff
@@ -8,6 +8,7 @@
 			<ResourceDictionary.MergedDictionaries>
 				<XamlControlsResources xmlns="using:Microsoft.UI.Xaml.Controls" />
 				<!--  Other merged dictionaries here  -->
+				<ResourceDictionary Source="../Whim.Bar/bar.defaults.xaml" />
```

Comment:
Is it possible for the `ResourceDictionary` be loaded into `App.xaml` during runtime as Whim starts up?

Reply:
> What would be the best place to put this? A new private void inside `App` that gets called from `App.StartWhim`?

`BarPlugin.PreInitialize` would be ideal.

---

> With this in mind, would it make sense to move the default `ResourceDictionary` outside of `/plugins/bar/`, say to `/resources/`?

This makes sense, let's go with this.

Reply:
My initial thought was to have different XAML files for each extension, but I think having a single file has its merits too - let's go with that.